modules/jaggery-apps/appmgt/src/site/pages/createapplication.jag

```diff
@@ -22,6 +22,10 @@ var site = require("/site/conf/site.json");
 
 (function () {
     var type = request.getParameter('type');
+    var apptype = request.getParameter('apptype');
+    var apptypeIconColor = request.getParameter('apptypeIconColor');
+    var apptypeIconImageClass = request.getParameter('apptypeIconImageClass');
+    var appTypeDisplayName = request.getParameter('appTypeDisplayName');

```

Comment:
Problem is whenever we redirected to this page, we have to set those parameters. I f we leave like this, when new element is introduced (per se "appTypeAnimation" :blush: ) we have to send it as this, when ever we call this page. So sending 'type' is enough  and all other things can be get from the type.